bin/nvcc_wrapper

```diff
@@ -90,6 +90,11 @@ replace_pragma_ident=0
 # Mark first host compiler argument
 first_xcompiler_arg=1
 
+# Allow for setting temp dir without setting TMPDIR in parent (see https://docs.olcf.ornl.gov/systems/summit_user_guide.html#setting-tmpdir-causes-jsm-jsrun-errors-job-state-flip-flop)
+if [[ ! -z ${_NVCC_WRAPPER_TMPDIR_OVERRIDE+x} ]]
+then
```

Comment:
````suggestion
if [[ ! -z ${NVCC_WRAPPER_TMPDIR+x} ]]; then
  temp_dir=${TMPDIR:-/tmp}
else
  temp_dir=${NVCC_WRAPPER_TMPDIR}
fi
````

I think there's some bash foo you could even do to make this a one-liner, which I don't remember right now.